README.md

```diff
@@ -276,6 +215,75 @@ the experiment.
 A list of know issues can be found [here](https://emdgroup.github.io/baybe/known_issues.html).
 
 
+## Advanced Installation

```

Comment:
I personally would prefer to hace this not after but still before the authors, but happy to hear other opinions here.

Reply:
id also prefer to have authors very last, but switching install and quickstart is fine with me

Reply:
Sounds good 👍 will correct today

Reply:
Fixed